src/acl/acl.internal.ts

```diff
@@ -463,3 +465,24 @@ export function internal_duplicateAclRule(sourceRule: AclRule): AclRule {
 
   return targetRule;
 }
+
+export function internal_setResourceAcl<
+  ResourceExt extends WithServerResourceInfo
+>(
+  resource: ResourceExt,
+  acl: WithResourceAcl["internal_acl"]
+): ResourceExt & WithResourceAcl;
+export function internal_setResourceAcl<
+  ResourceExt extends WithServerResourceInfo
+>(
+  resource: ResourceExt,
+  acl: WithFallbackAcl["internal_acl"]
+): ResourceExt & WithFallbackAcl;
```

Comment:
Given that we've so far been referring to a Resource's parent's ACL as "fallback ACL" and a Resource's own ACL as a "Resource ACL", and given that this function accepts both, should we just call it `internal_setAcl`?

Reply:
I wasn't really happy with the previous name for that reason, so very good suggestion 03346bc8be43d54a698f05321c203c4160d73b6f